dist/index.js

```diff
@@ -12114,6 +12114,7 @@ async function hostBuild(maturinRelease, args) {
     const sccache = core.getBooleanInput('sccache');
     const isUniversal2 = args.includes('--universal2') || target === 'universal2-apple-darwin';
     core.startGroup('Install Rust target');
+    await exec.exec('rustup', ['update']);
```

Comment:
I don't quite like doing this unconditionally, I think we only need to do this when Rust toolchain is not specified or set to `stable`/`beta`/`nightly` while the currently installed version < MSRV specified in `Cargo.toml`.

Reply:
That might break many repos upgrading to edition2024 with the default GitHub action settings. We might also need to update the README.

How about moving it to the next `if (rustToolchain)` block? So it will always update the toolchain if specified. My concern is that `MSRV` is not a compulsory configuration while it's common to use the latest stable version in dev.

Reply:
> How about moving it to the next `if (rustToolchain)` block?

Sounds better and it should also pass the rust toolchain argument.